tools/slang-unit-test/unit-test-secure-crt.cpp

```
@@ -0,0 +1,496 @@
+    // null mode
+    {
+        FILE* f = nullptr;
+        errno = 0;
+        SLANG_CHECK(fopen_s(&f, kNullDevice, nullptr) == EINVAL);
+        SLANG_CHECK(errno == EINVAL);
+    }
+    // nonexistent file
+    {
+        FILE* f = nullptr;
+        int result = fopen_s(&f, "/nonexistent/path/file.txt", "r");
+        SLANG_CHECK(result != 0);
```

Cppcheck (2.20.0)

[style] The function 'FastFloor' is never used.

(unusedFunction)